### ISSUE TITLE
Video Stats not reporting

### ISSUE THREAD

**Issue by dmkjr** [kylegilman/video-embed-thumbnail-generator#397]
I just noticed that although all of our videos are displayed on our site using your plugin, it appears to not be counting the stats.  I'm not sure if it's because our media isn't played from our servers?  We use W3 Offload to offload all media to Amazon S3.  

Thanks for any help provided.

**Comment by kylegilman** [comment]
Are you looking for stats in the WordPress admin area or Google Analytics? Can you post a link to your site so I can troubleshoot it?

**Comment by dmkjr** [comment]
Wordpress Admin area I see a section now from your plugin that lets me know how many video views we have on each video.  But, it's just showing zero even though there are thousands of visitors on our site.  I just thought perhaps it had something to do with us offloading the media.

https://cityofhanahan.com

**Comment by dmkjr** [comment]
Few videos are loaded at https://cityofhanahan.com/notices/

**Comment by kylegilman** [comment]
Yeah, the plugin is acting as if the video is not in your WordPress Media Library, but it looks to me as if it is. I think the problem is that you have versioning turned on in S3, which adds additional text to your URL and confuses my plugin's systems. For example, one of your videos has this URL `https://cdn2.cityofhanahan.com/wp-content/uploads/2018/02/23204223/FD-CPR-VIDEO.mp4` which is most likely recorded in your database as `wp-content/uploads/2018/02/FD-CPR-VIDEO.mp4`. 

What is the shortcode you're using to embed that video at https://cityofhanahan.com/notices/hands-free-cpr/ ?

**Comment by dmkjr** [comment]
[KGVID]https://cdn2.cityofhanahan.com/wp-content/uploads/2018/02/23204223/FD-CPR-VIDEO.mp4[/KGVID]

**Comment by dmkjr** [comment]
I can cut off versioning in S3 using the WP Offload settings.

**Comment by kylegilman** [comment]
I know you can turn off versioning in S3, but I'm worried that it will break all your embedded videos since the shortcode has the extra text in it and turning off versioning might make those URLs invalid. If you don't have too many videos already embedded, you can just manually edit the shortcode to remove the extra numbers, but if you have a lot it might be too much work. 

I can try to come up with a solution to accommodate versioning but I don't have much time to work on the plugin right now so I don't know how quickly I'd be able to implement it.

**Comment by dmkjr** [comment]
Thank you for your help!  The best video plugin I have found to date.
